packages/wallets/src/components/WalletListCard/WalletListCard.tsx

```diff
@@ -1,35 +1,32 @@
 import React from 'react';
-import { THooks } from '../../types';
+import { useActiveWalletAccount } from '@deriv/api-v2';
 import { WalletCardIcon } from '../WalletCardIcon';
 import WalletListCardDetails from '../WalletListCardDetails/WalletListCardDetails';
 import './WalletListCard.scss';
 
-type TProps = {
-    balance: THooks.WalletAccountsList['display_balance'];
-    currency: THooks.WalletAccountsList['wallet_currency_type'];
-    isActive: THooks.WalletAccountsList['is_active'];
-    isDemo: THooks.WalletAccountsList['is_virtual'];
-    loginid: THooks.WalletAccountsList['loginid'];
-    onAccountSelect: (loginid: string) => void;
-};
+const WalletListCard = () => {
+    const { data: activeWallet } = useActiveWalletAccount();
+
+    const currency = activeWallet?.wallet_currency_type || 'USD';
```

Comment:
do we need to wait for other endpoionts like useCurrency and useCurrencyConfig to make sure this is available and USD won't flicker for a brief moment? (might be scary to users if it flickers and might be super annoying for all sort of component tests)

Reply:
I believe yes @wojciech-deriv. `website_status` does take some time but not as long as authorize sometimes. But there is some minor delay